c2corg_api/views/user.py

```diff
@@ -113,6 +122,24 @@ def validate_forum_username(request, **kwargs):
             request.errors.add('body', attrname, res)
 
 
+def validate_username(request, **kwargs):
+    """
+    Check that the username is not an email,
+    or that it is the same as the actual email.
+    """
+    attrname = 'username'
+    if attrname in request.json and 'email' in request.json:
```

Comment:
That's not very consistent to use a variable for username but not for email. Why not simply write `'username'` everywhere as you did for `'email'`?
That would be more readable I think.

Reply:
I tried to be a bit consistent with other cases, but agreed